CNAME

```diff
@@ -1 +0,0 @@
-www.cicsoft.tech
```

Comment:
Never delete the CNAME file it is used to identify our domain using Github Pages